core/dbt/parser/schema_yaml_readers.py

```diff
@@ -117,6 +117,12 @@ def parse_exposure(self, unparsed: UnparsedExposure) -> None:
                 f"Calculated a {type(config)} for an exposure, but expected an ExposureConfig"
             )
 
+        tags = sorted(set(self.project.exposures.get("tags", []) + unparsed.tags + config.tags))
+        meta = {**self.project.exposures.get("meta", {}), **unparsed.meta, **config.meta}
```

Comment:
Sweet, if my understanding is corrected, this ordering is such that  `config.x` will take precedence over `unparsed.x` for any key that is defined in both.

Reply:
yes, correct.